dotnet/Services/ProductReviewService.cs

```diff
@@ -318,6 +318,11 @@ public async Task<ReviewsResponseWrapper> GetReviewsByProductId(string productId
                 searchQuery = $"&_keyword={searchTerm}";
             }
 
+            AppSettings settings = await GetAppSettings();
+            if (settings.RequireApproval)
+            {
+                searchQuery = $"{searchQuery}&approved=true";
```

Comment:
I think you may need a `_where` argument here, like:
`searchQuery = $"{searchQuery}&_where=approved=true";`

Reply:
this _where is not necessary as i need only a simple one field filter according whit the documentation here: https://developers.vtex.com/vtex-rest-api/reference/search#simple-filter

Reply:
What if `searchQuery` already has a value? Will it work if you do something like `&_keyword=good&approved=true` ?